app/_locales/es/messages.json

```diff
@@ -929,5 +929,579 @@
+  },
+  "notFound": {
+    "message": "No se localizó"
+  },
+  "noWebcamFoundTitle": {
+    "message": "No se localizó al webcam"
+  },
+  "noWebcamFound": {
+    "message": "No se localizó al webcam de tu ordenador. Favor de volver a intentar."
+  },
+  "ofTextNofM": {
+    "message": "de"
+  },
+  "onlySendTokensToAccountAddress": {
+    "message": "Mandar solamente $1 a la dirección de una cuenta de Ethereum.",
+    "description": "muestra el símbolo del token"
+  },
+  "openInTab": {
+    "message": "Abrir en una pestaña"
+  },
+  "orderOneHere": {
+    "message": "Comprate un Trezor o Ledger y guarda tus fondos en almacenamiento frío"
+  },
+  "origin": {
+    "message": "Orígen"
+  },
+  "outgoing": {
+    "message": "De salida"
+  },
+  "parameters": {
+    "message": "Parámetros"
+  },
+  "originalTotal": {
+    "message": "Total Original"
+  },
+  "password": {
+    "message": "Contraseña"
+  },
+  "pending": {
+    "message": "pendiente"
+  },  
+  "popularTokens": {
+    "message": "Tokens Corrientes"
+  },
+  "prev": {
+    "message": "Prev"
+  },
+  "primaryCurrencySetting": {
+    "message": "Moneda Principal"
+  },
+  "primaryCurrencySettingDescription": {
+    "message": "Seleccionar nativa para prioritizar el que se muestren los valores en la moneda nativa de la cadena (p.ej. ETH). Seleccionar Fíat para prioritzar el que se muestren los valores en la moneda fíat seleccionada."
+  },
+  "queue": {
+    "message": "Cola"
+  },
+  "rejectAll": {
+    "message": "Rechazar a Todos"
+  },
+  "rejectTxsN": {
+    "message": "Rechazar a transacciones de $1"
+  },
+  "rejectTxsDescription": {
+    "message": "Está al punto de rechazar transacciones de $1 en lote."
+  },
+  "reset": {
+    "message": "Reiniciar"
+  },
+  "resetAccountDescription": {
+    "message": "Reiniciar tu cuenta borrará tu historial de transacciones."
+  },
+  "restoreAccountWithSeed": {
+    "message": "Restaurar tu Cuenta con Frase Semilla"
+  },
+  "requestsAwaitingAcknowledgement": {
+    "message": "peticiones pendientes de reconocimiento"
+  },
+  "restore": {
+    "message": "Restaurar"
+  },
+  "revealSeedWordsTitle": {
+    "message": "Frase Semilla"
+  },
+  "revealSeedWordsDescription": {
+    "message": "Si en algún momento cambias de navegador o de ordenador, necesitarás esta frase semilla para acceder a tus cuentas. Guárdatela en un lugar seguro y secreto."
+  },
+  "revealSeedWordsWarningTitle": {
+    "message": "NO compartas esta frase con nadie!"
+  },
+  "remove": {
+    "message": "borrar"
+  },
+  "removeAccount": {
+    "message": "Borrar cuenta"
+  },
+  "removeAccountDescription": {
+    "message": "Se borrará esta cuenta de tu monedero. Favor de aseurar que tengas la frase semilla o clave personal original para esta cuenta importada antes de seguir adelante. Podrás importar o crear cuentas de nuevo del menu desplegable de cuentas."
+  },
+  "readyToConnect": {
+    "message": "¿List@ para conectar?"
```

Comment:
Listo/a

Reply:
Hahahaa. I was wondering if I should go with "Listx" lol. But if your vote is for listo/a I'm good with it.

Reply:
In fact, I'm not sure at all about that either. But I felt that "o/a" is inclusive and classic at the same time.